src/common/containers/UserForm/index.jsx

```diff
@@ -76,18 +76,18 @@ function mapStateToProps(state, props) {
   const {identifier} = props.params
   const {app, users, phases, auth} = state
   const user = findAny(users.users, identifier, ['id', 'handle'])
+  const phase = user && user.phase ? user.phase : {}
 
-  const sortedPhases = Object.values(phases.phases).sort((p1, p2) => p1.number - p2.number)
-  const sortedPhaseOptions = [...sortedPhases.map(phaseToOption), {value: null, label: 'No Phase'}]
+  const sortedPhaseOptions = phasesToOptions(phases.phases)
 
-  let formType = FORM_TYPES.UPDATE
-  if (identifier && !user && !users.isBusy) {
-    formType = FORM_TYPES.NOT_FOUND
-  }
+  const formType = identifier && user && !users.isBusy ?
```

Comment:
Is this a bug? Previous version has the condition `identifier && !user && !users.isBusy`, but this version omits the ! in front of `user`.

Reply:
Updated